docs/troubleshooting/not-loading.md

```diff
@@ -55,16 +55,53 @@ How to start a simple HTTP server:
 ### Symptom #2: The app says "Please wait..." forever
 
 If when you try to load your app in a browser you see a blue box in the center
```

Comment:
'If when' is a bit awkward, but not enough to hold this up. LGTM

Reply:
Thanks! I'm trying to set up a better process so we don't have to do these last-minute changes, so I'm hoping this is the last time this will happen.

In the future, all changes (including changes to docs) will be properly reviewed before release, even if it means delaying the release.